tests/md-nacl.rs

```diff
@@ -1,7 +1,7 @@
 // Lumol, an extensible molecular simulation engine
 // Copyright (C) Lumol's contributors — BSD license
 
-//! Testing physical properties of a NaCl crystal
+//! Testing physical properties of a sodium chloride crystal
```

Comment:
Is that really better ? I mean I get NaCl, sodium chloride I have to think about it ^^